sdk/js/src/examples/simpleSearcherSvm.ts

```diff
@@ -89,12 +92,12 @@ export class SimpleSearcherLimo {
   }
 
   /**
-   * Generates a bid for a given opportunity.
+   * Generates a bid for a given limo opportunity.
    * The transaction in this bid transfers assets from the searcher's wallet to fulfill the limit order.
    * @param opportunity The SVM opportunity to bid on.
    * @returns The generated bid object.
    */
-  async generateBid(opportunity: OpportunitySvm): Promise<BidSvm> {
+  async generateBidLimo(opportunity: OpportunitySvmLimo): Promise<BidSvm> {
```

Comment:
nit, i still prefer limo and swap examples to live in two different files bc they are completely different functions